service/controller/v21/cloudconfig/master_template.go

```diff
@@ -44,9 +44,7 @@ func (c *CloudConfig) NewMasterTemplate(ctx context.Context, customObject v1alph
 		params = k8scloudconfig.DefaultParams()
 
 		params.Cluster = customObject.Spec.Cluster
-		params.DisableCoreDNS = true
 		params.DisableEncryptionAtREST = true
-		params.DisableIngressController = true
```

Comment:
Only non-vendor change is removing k8scc params. Already tested manually so will merge once e2e is green.